python/tests/conftest.py

```diff
@@ -50,7 +51,7 @@ def spark(tmp_path_factory) -> SparkSession:
             "spark.executor.extraJavaOptions",
             "-Dio.netty.tryReflectionSetAccessible=true",
         )
-        .config("spark.rikai.cacheUri", str(tmp_path_factory.mktemp("data")))
+        .config(CONF_RIKAI_CACHEURI, str(tmp_path_factory.mktemp("data")))
```

Comment:
is this config needed in notebook as well?

Reply:
it would be yes